### PR TITLE
Add missing tools

### LINKED ISSUE #137: Kali Tools Request
# 💡 Summary #

Request for missing and additional tools to be added to Kali AMI build

## Motivation and context ##

Per discussions with @jsf9k there appear to be some tools that weren't present on the latest AMI build that need to be explicitly added. This issue is to provide a listing of the tools that we noticed were missing including some additional tools that we are consistently using on our engagements. 

## Implementation notes ##

We would like to include the following tools and environment variables on the Kali instances:

- nmap 
- vim
- [Feroxbuster](https://github.com/epi052/feroxbuster.git)
- [Subfinder](https://github.com/projectdiscovery/subfinder.git)
- [jq](https://github.com/stedolan/jq.git)
- [assetfinder](https://github.com/tomnomnom/assetfinder.git)
- [cargo](https://github.com/rust-lang/cargo.git)
- [ripgen](https://github.com/resyncgg/ripgen.git)
- [waybackurls](https://github.com/tomnomnom/waybackurls.git)
- [gau](https://github.com/lc/gau.git)
- GO environment variables setup as shown [here](https://rafaelhart.com/2020/03/installing-golang-on-kali-linux/)

## Acceptance criteria ##

How do we know when this work is done?

- [x] All tools and environment variables listed above have been installed along with their dependencies.

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request adds some tools to the Kali AMI that were requested by the users.

## 💭 Motivation and context ##

Resolves #137.

## 🧪 Testing ##

All automated tests pass.  I also created a staging AMI with these changes and verified that it functions as expected.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All new and existing tests pass.

## ✅ Pre-merge checklist ##

- [x] Finalize version.

## ✅ Post-merge checklist ##

- [x] Create a release.

### CODE REVIEW

**Review by dav3r — APPROVED** [review]
Looks great - thanks for taking care of this!

**Comment by jmorrowomni** [comment]
lgtm

File: src/assessor_tools.yml [hunk 1]
```diff
@@ -389,21 +403,69 @@
...
           - veil
+          - vim
+
+    # In order for the VNC user to make full use of Go, we need to add
+    # $p/bin to the PATH for all paths p in $GOPATH.
+    #
+    # See #137 for more details.
+    - name: Add GOPATH binary paths to PATH for VNC user
+      ansible.builtin.blockinfile:
+        # The jinja2 comment above changes the Jinja2 templating to
```
Comment (mcdonnnj) [inline src/assessor_tools.yml]: Unless I've misunderstood something.

```suggestion
        # The jinja2 comment below changes the Jinja2 templating to
```

**Review by mcdonnnj — APPROVED** [review]
Looks about right to me. One suggestion about a comment for you but otherwise ✔

**Pull request merged**